packages/@aws-cdk/integ-runner/lib/runner/private/logger.ts

```diff
@@ -0,0 +1,20 @@
+import { Writable } from 'stream';
```

Comment:
I  took this file from `packages/aws-cdk`. It might be better if we had a shared library we could put stuff like this.

Reply:
Is there anything in `core` that we could reuse?